bin/cmd.js

```diff
@@ -1,334 +1,243 @@
 #!/usr/bin/env node
 'use strict'
 
-const clivas = require('clivas')
+// #region Variables
```

Comment:
?

Reply:
Replaced clivas with chalk+console.log. Clivas is old and broken, it caused the terminal to print the last line multiple times (if 
 I remember correctly). Chalk is well maintained and `console.log` is native 😃

Reply:
I'm wondering about what #region is

Reply:
Oh, lol. It allowes me to collapse parts in vscode.